assets/src/edit-story/app/story/useStoryReducer/reducers/addPage.js

```diff
@@ -45,10 +47,12 @@ function addPage(state, { page, position }) {
 
   const { id } = page;
 
-  // Ensure new page has elements array and background id
+  // Ensure new page has at least one element
+  if (!page.elements?.length >= 1) {
+    return state;
```

Comment:
Is this right? Would we want to add the element if it doesn't exist?

Reply:
It should never not exist. If you create the page via `addPage`, it'll come with one and you can never delete it. So it shouldn't happen.